README.md

```diff
@@ -66,34 +66,40 @@ Install the ``python3-routes`` package:
 
 These instructions are meant for developers who are compiling the code for
 development and testing. To build binaries that are suitable for installation
-we recommend that you build .deb or .rpm packages, or refer to ``ceph.spec.in``
-or ``debian/rules`` to see which configuration options are specified for
-production builds.
+we recommend that you build `.deb` or `.rpm` packages, or refer to
+``ceph.spec.in`` or ``debian/rules`` to see which configuration options are
+specified for production builds.
 
-Build instructions:
+To build ceph, make sure that you are in the top-level `/ceph` directory, and
+run the following commands:
 
 	./do_cmake.sh
 	cd build
 	ninja
 
-``do_cmake.sh`` defaults to creating a debug build of Ceph that can be up to 5x
-slower with some workloads. Pass ``-DCMAKE_BUILD_TYPE=RelWithDebInfo`` to
+``do_cmake.sh`` by default creates a "debug build" of Ceph, which can be up to
+5x slower with some workloads. Pass ``-DCMAKE_BUILD_TYPE=RelWithDebInfo`` to
 ``do_cmake.sh`` to create a non-debug release.
 
 The number of jobs used by `ninja` is derived from the number of CPU cores of
```

Comment:
I'd love to see a sentence about what `ninja` is.  I have no idea.

Reply:
I've added a sentence. Good looking out. This is exactly the kind of thing I want to add to the docs.